src/browser/pdf.ts

```diff
@@ -0,0 +1,15 @@
+import * as puppeteer from 'puppeteer';
+
+// Allow setting the more esoteric PDF options via URL parameters 
+export function getPDFOptionsFromURL(url: string): puppeteer.PDFOptions {
+  const urlParams = new URLSearchParams(url);
+  return {
+    landscape: urlParams.get('pdf.landscape') === 'true',
+    format: (urlParams.get('pdf.format') as puppeteer.PaperFormat) ?? 'Letter',
+    preferCSSPageSize: urlParams.get('pdf.preferCSSPageSize') !== 'false', // defaults true
+    omitBackground: urlParams.get('pdf.omitBackground') !== 'false', // defaults true,
+    printBackground: urlParams.get('pdf.printBackground') === 'true', 
+    pageRanges: urlParams.get('pdf.pageRanges') ?? undefined,
+    tagged: urlParams.get('pdf.tagged') === 'true',
+  };
+}
```

Comment:
This allows for changing properties via URL eg:

http://localhost:3000/render/d/MP-Di9F7k/candlestick?orgId=1&kiosk=tv&from=1626214470464&to=1626216259472&theme=light&encoding=pdf&pdf.landscape=true&pdf.format=Legal